File: app/src/main/java/com/neome/java/api/meta/base/dto/EntVdErdDia.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Symbol;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;
import com.neome.java.api.meta.base.Types.MetaIdVdErdDia;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class EntVdErdDia extends EntVdDia
{
  @Nullable
  public Map<MetaIdSpreadsheet, EntVdErdEntity> entityMap;

  @Nullable
  public String label;

  public MetaIdVdErdDia metaId;

  public Symbol name;

  @Nullable
  public Map<MetaIdField, EntVdErdRef> refMap;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdErdDia.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdDia
import com.neome.api.meta.base.dto.EntVdErdEntity
import com.neome.api.meta.base.dto.EntVdErdRef
import java.util.Map
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdSpreadsheet
import com.neome.api.meta.base.Types.MetaIdVdErdDia
import com.neome.api.meta.base.Symbol

interface EntVdErdDia : EntVdDia
{
  val entityMap: Map<MetaIdSpreadsheet, EntVdErdEntity>?
  val label: String?
  val metaId: MetaIdVdErdDia
  val name: Symbol
  val refMap: Map<MetaIdField, EntVdErdRef>?
}